python/cugraph/cugraph/tests/centrality/test_betweenness_centrality.py

```diff
@@ -507,6 +507,7 @@ def test_betweenness_invalid_dtype(
 @pytest.mark.parametrize("graph_file", utils.DATASETS_SMALL)
 @pytest.mark.parametrize("directed", DIRECTED_GRAPH_OPTIONS)
 @pytest.mark.parametrize("edgevals", WEIGHTED_GRAPH_OPTIONS)
+@pytest.mark.filterwarnings("ignore::DeprecationWarning")
```

Comment:
I think this is going to filter out *all* `DeprecationWarning`s, including those raised by upstream libraries that we need to know about (eg. `cudf`).  I believe you should be able to capture just the warnings we raise though by providing a different string ([link](https://docs.pytest.org/en/7.1.x/how-to/capture-warnings.html#pytest-mark-filterwarnings)).
But maybe instead of adding all these markers, can you instead edit our [pytest.ini](https://github.com/rapidsai/cugraph/blob/branch-24.08/python/cugraph/pytest.ini)? There's a section where `DeprecationWarnings` are configured to be errors followed by all the deprecations that should be ignored.

Reply:
it will and that could be an issue, but it was the only way to capture the deprivation that we added.  Now there are still test that leverage nx-cugraph directly that will capture NetworkX deprication

Reply:
the "ignore" is added to just those tests related to accepting a NetworkX object.